SerhiiPapusha/HW6/6.1.py

```diff
@@ -0,0 +1,6 @@
+divisible_by_two = [x for x in range(1, 10) if x % 2 == 0]
+print(divisible_by_two)
+divisible_by_three= [x for x in range(1, 10) if x % 3 == 0]
```

Comment:
according to the condition, it was necessary to write the code like this
divisible_by_three = [ x for x in range(1, 11) if x % 2 == 1 and x % 3 == 0 ]